source/common/ssl/context_impl.cc

```diff
@@ -92,8 +98,6 @@ ContextImpl::ContextImpl(ContextManagerImpl& parent, Stats::Scope& scope, Contex
     }
   }
 
-  SSL_CTX_set_options(ctx_.get(), SSL_OP_NO_SSLv3);
```

Comment:
This is because v3 is gone now and this is obsolete?

Reply:
No, it's because we set minimum protocol version to TLSv1 or higher via `SSL_CTX_set_min_proto_version()`, so BoringSSL won't negotiate SSLv3 (and it was disabled by default for a while anyway).